File: AndroidStudioProjects/FirstApp/app/src/main/java/com/ecrinaybay/firstapp/MainActivity.java
package com.ecrinaybay.firstapp;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.ImageView;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    public void changeImage(View view) { //Büyük View hangi sınıftan üretildiği küçük view bu sınıftan üretilen objectin ismi
        ImageView imageView = findViewById(R.id.imageView);
        imageView.setImageResource(R.drawable.manga2);
    }



}

File: AndroidStudioProjects/TestIntro/app/src/main/java/com/ecrinaybay/testintro/MainActivity.kt
package com.ecrinaybay.testintro

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val tax = Tax()
        val netIncome = tax.calculateIncome(100.0,0.1)
        val netTax = tax.calculateTax(100.0,0.1)



    }
}
































































